template/build/webpack.dev.conf.js

```diff
@@ -65,7 +64,11 @@ const devWebpackConfig = merge(baseWebpackConfig, {
         ignore: ['.*']
       }
     ])
-  ]
+  ],
+  optimization: {
```

Comment:
This is not necessary. In the `mode: 'development'` is the default configuration.

Reply:
You're right about `namedModules`. I'm removing it.

However, the [docs](https://webpack.js.org/configuration/optimization/#optimization-noemitonerrors) for `noEmitOnErrors` seem to indicate that it does not have a default value, so I'm leaving it for now.